src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="image"
+        src="images/imac.jpeg"
+        alt="imac"
+      />
+
+      <div class="content__description">
+        <div class="description--name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="description--code">Product code: 195434</div>
+      </div>
+      <div class="content__stars">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="reviews">Reviews: 5</div>
+      </div>
+      <div class="content__price">
+        <div class="price--label">Price:</div>
+        <div class="price--numbers">$2,199</div>
+      </div>
+      <!-- <button class="button">
+        <a
+          class="button__link"
+          href="buy"
+        >
+          BUY
+        </a>
+      </button> -->
+      <a
+        class="button__link"
+        href="buy"
+        data-qa="hover"
+      >
+        <button class="button">BUY</button>
+      </a>
```

Comment:
It's not recommended to place a `<button>` element inside an `<a>` tag. This can cause accessibility issues and is semantically incorrect. Consider using either a button or a link, depending on the intended action. If the action is to navigate to another page, use an `<a>` tag styled as a button.